toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -42,52 +47,53 @@ cd gcc-9.1.0
 ln -s ../mpfr-4.0.1 mpfr
 ln -s ../gmp-6.1.2 gmp
 ln -s ../mpc-1.1.0 mpc
-
-# Patch GCC to avoid an error where PATH_MAX isn't defined
-patch -p1 < "${scriptDir}/pathmax.patch"
```

Comment:
Awesome! PATH_MAX patch no longer needed 😃 I'm guessing that the sysroot separation helped resolve the proper native vs target headers used by gcc?

Reply:
Also, please remove the pathmax.patch file as well

Reply:
Yes the headers are now on the right path so we don't need this anymore :). I removed the patch